docs/jwst/extract_1d/description.rst

```diff
@@ -302,3 +301,43 @@ non-differentiated error term.  Note that while covariance is also extremely imp
+and lower layers, encompassing the detector substrate and the infrared-active layer. Since the thickness of the substrate
+is not the same in the SW and LW detectors, the fringe frequency will differ in the two detectors. Up to 16 microns, this
+fringe is produced by the anti-reflection coating and  pixel metalization etalons, whereas above 16 microns it is
+produced by the anti-reflection coating and  bottom contact etalon, resulting in a different fringe frequency.
+
+The JWST pipeline contains multiple steps to mitigate the impact of fringing on science spectra and these steps generally
+suffice to reduce the fringe signal to below a few percent of the target flux.
+
+The first correction is applied by default in the ``fringe_step`` in the :ref:`calwebb_spec2 <calwebb_spec2>` pipeline
+and  consists in dividing the uncalibrated "rate" image by a static fringe flat constructed from observations of a
+bright source that fills the entire MRS field of view. This step is described in the :ref:`fringe <fringe_step>` step.
+This step generally does a good job of removing the strongest
+fringes from an astronomical scene, particularly for nearly-uniform extended sources. Since the fringe
+signal is different for point sources however (and varies as a function of the location of a point source within the FOV)
+the static fringe flat cannot fully correct such objects and the default high level data products will therefore still
+show appreciable fringes.
+
+
+The pipeline also includes two optional residual fringe correction steps whose purpose is to find and remove signals
+whose periodicity is consistent with known fringe frequencies (set by the optical thickness of the detectors and dichroics)
+using a Lomb-Scargle periodogram. The number of fringe components to be removed is governed by a Bayesian evidence calculation.
+The first of these residual fringe correction steps is a 2-D correction that can be applied to the flux-calibrated detector data
+in the ``residual_fringe_step``. This step is part of the :ref:`calwebb_spec2 <calwebb_spec2>` pipeline, but currently
+it is skipped by default. For more information see :ref:`residual_fringe <residual_fringe_step>`.
+
+The pipeline also can apply a 1-D residual fringe correction. This correction is only for MIRI MRS data and 
+can be turned on by setting the optional parameter ``extract_1d.ifu_rfcorr = True``  in the ``extract_1d`` step. 
+Empirically, the 1-D correction step has been found to both work better than the 2-D correction step if is applied to per-band spectra.
+When using the ``extract_1d`` step to apply a 1-D residual fringe
+correction the correction is applied during the extraction of spectra from the IFU cube. The 1D residual fringe code can be called outside the
+pipeline to correct an extracted spectrum. If running outside the pipeline the correction works best on single-band cubes and the channel of
+the data must be given. The steps to run this correction outside the pipeline are:
```

Comment:
random question, but if the channel truly **must** be given, shouldn't we make the channel parameter non-optional in the function?

Reply:
The program figures out what channel the data before calling fit_residual_fringes_1d.   So when running the 1d extraction in the pipeline the user does not need to supply it. However if running outside the pipeline (and a lot of people do their own extraction method) then the user does need to supply the channel because the code that figures that out is not part of fit_residual_fringes_1d.

Reply:
but in both cases, the channel has to be known before calling the function, right?  The default is meaningless for the pipeline, which figures out what value to pass in beforehand, and it is meaningless to the user, who must specify the channel per the documentation.  Or maybe I'm still misunderstanding.

This is obviously a very small nit-pick and I assume it wouldn't be in this PR that is completely docs-focused anyway, I was just curious.

Reply:
When running the code in the pipeline the code figures out the channel. When running outside the pipeline the user has to supply the channel.
Does that make sense ?

Reply:
there is no case where the function gets run and we don't care what the channel is, or we think, "yeah let's just set that to whatever the default is, it doesn't matter"